exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/statistics/StatisticsMergeBatch.java

```diff
@@ -92,11 +92,14 @@
   private int schema = 0;
   private int recordCount = 0;
   private List<String> columnsList = null;
+  private double percent = 100;
```

Comment:
Here, percent is being correctly treated as up to 100 whereas in another place I saw it was being treated as a fractional number.

Reply:
Done